lib/flipper/api/v1/decorators/actor.rb

```diff
@@ -0,0 +1,39 @@
+module Flipper
+  module Api
+    module V1
+      module Decorators
+        class Actor < SimpleDelegator
+          # Public: the actor and feature.
+          attr_reader :actor, :features
+
+          def initialize(actor, features)
+            @actor = actor
+            @features = features
+          end
+
+          def as_json
+            {
+              'flipper_id' => actor.flipper_id,
+              'features' => features_data,
+            }
+          end
+
+          private
+
+          def features_data
+            features.map do |feature|
+              {
+                'feature' => feature.name,
+                'enabled' => enabled?(feature),
+              }
+            end
+          end
+
+          def enabled?(feature)
```

Comment:
Does this method help us at all? I wonder if we could just inline it since it is only used once and the contents of it are one very simple method call.